.github/auto-merge.yml

```diff
@@ -0,0 +1,83 @@
+- match:
+    dependency_name: FluentValidation
+    update_type: all
+
+- match:
+    dependency_name: Fody
+    update_type: all
+
+- match:
+    dependency_name: GitVersionTask
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.AspNetCore.Server.Kestrel
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.AspNetCore.StaticFiles
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.CodeAnalysis.CSharp.Workspaces
```

Comment:
We can't automerge this one. We need to stay on an older version to ensure the analyzers built into the NSB package can [support older versions of VS](https://docs.microsoft.com/en-us/visualstudio/extensibility/roslyn-version-support?view=vs-2019).